blocks/hero/hero.js

```diff
@@ -41,11 +41,12 @@ export default async function decorate(block) {
         getMetadata('author'),
         ' •',
       ) : '',
-      getMetadata('article:published_time') ? span(
+      getMetadata('published-time') ? span(
         { class: ['media-blend__date'] },
-        formatDate(getMetadata('article:published_time')),
+        formatDate(getMetadata('published-time')),
         getMetadata('article:read_time') ? ' •' : '',
       ) : '',
+      // TODO this is wrong we don't have read time in metadata
```

Comment:
on the current news.sap.com, it seems to sit in `<meta name="twitter:data2" content="4 minutes" />`

Reply:
Correct we never used `article:read_time` so far, I will change it.